.gitignore

```diff
@@ -3,6 +3,7 @@
 /go-bindata
 /machine*
 /bin
+/vendor
```

Comment:
To just keep `vendor-release` in git? `build` script would copy the `vendor-release` to `vendor`.